### DISCUSSION TITLE
Where can I download the 7 mm motor mounts structure file?

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1569] [General]
Where can I download the 7 mm motor mounts structure file? We hope to make some structural improvements.

![image](https://github.com/user-attachments/assets/13ed47cc-01a7-4dfe-b039-bb335330064d)

**Comment by knmcguire** [comment]
Hi! we have a repo with some .stl mounts but those aren't really up to date: https://github.com/bitcraze/bitcraze-mechanics

We will discuss it this week if we can release the latest as well, since we do have some restrictions on what we can release. I'll update you as soon as I know more.

**Reply by justinleeyang** [reply]
Thanks

**Reply by Bananazooka** [reply]
Hi Justin, 
Right now, [those files ](https://github.com/bitcraze/bitcraze-mechanics/tree/master/motor-mounts/cf2-mount-openscad)on the bitcraze mechanics repo are the best options - those are the only .stl files we have that are 3D printable. They are a little outdated but it's the best we can do.

**Reply by justinleeyang** [reply]
so don't provide the motor mounts file ?
https://store.bitcraze.io/products/crazyflie-2-0-4-x-spare-7-mm-motor-mounts

**Reply by justinleeyang** [reply]
It seems that this motor mount is not sold in China. Can you provide the files so that we can make it ourselves?

**Comment by Bananazooka** [comment]
You can buy motor mounts through Seeed - they're out of stock right now but you can maybe contact them to see when they plan on restocking. Otherwise, you may go through our shop to buy them! Don't hesitate to email sales@bitcraze.se if you run into problems.
